netmiko/ssh_autodetect.py

```diff
@@ -167,6 +167,12 @@
         "priority": 99,
         "dispatch": "_autodetect_std",
     },
+    "yamaha": {
+        "cmd": "show environment",
+        "search_patterns": [r"RTX", r"NVR", r"FWX"],
```

Comment:
This is probably too general...would probably need an additional word like "Yamaha" in each of these search patterns (obviously has to match the output the device sends back for "show environment").